CONTRIBUTING.md

```diff
@@ -42,17 +46,20 @@ To contribute to the HADDOCK3's Python shell, follow these steps:
 
 5.  You can work on these `tox` tests until they all pass green before
 submitting your PR.
-6.  We also have an `examples` folder with `test` cases that you can run
+6.  We also have an `examples` folder with test cases that you can run
```

Comment:
Add some explanation that the tests perform a reduced sampling and are only meant to test the machinery. The example folder also contain full config files for production runs.

Reply:
And may-be we should give an estimate of the time required to run those, e.g. on an 8 core computer (i.e. using 7 cores). Could use my laptop for this